components/layouts/Menu.vue

```diff
@@ -0,0 +1,164 @@
+<template>
+  <div class="menu">
```

Comment:
[It is recommended](https://vuejs.org/v2/guide/components-registration.html#Component-Names) that all the components contain a hyphen. We are using UpperCamelCase for our components which means we should use multi-word names. This is because HTML reserves its right for creating new tags, and those will be one word.

I'm not complaining about the components that already are named with one word (although [I filed #585 to take care of it at some point) but I prefer not to lose those well named. Nevertheless, I agree that `QiskitOrgMenu` is useful no more and we can replace it with `TheMenu` (as [I just discover it is recommended](https://vuejs.org/v2/style-guide/#Single-instance-component-names-strongly-recommended) by Vue.js) or something that points this is the main menu and not any other menu.